infra/libfuzzer-pipeline.groovy

```diff
@@ -65,6 +63,7 @@ def call(body) {
           writeFile file: "$wsPwd/${sanitizer}.rev", text: revText
           echo "revisions: $revText"
 
+          def dockerfile = "$workspace/oss-fuzz/$projectName/Dockerfile"
```

Comment:
This won't work for projects that decide to keep their fuzzers and Dockerfiles in the source tree, right?

I have a small example: https://github.com/mikea/ossfuzz-example/blob/master/fuzz/Jenkinsfile